typescript/infra/config/environments/mainnet2/igp.ts

```diff
@@ -24,24 +24,21 @@ function getGasOracles(local: MainnetChains) {
   );
 }
 
-export const igp: ChainMap<OverheadIgpConfig> = objMap(
-  owners,
-  (chain, owner) => {
-    return {
-      owner,
-      oracleKey: DEPLOYER_ADDRESS,
-      beneficiary: KEY_FUNDER_ADDRESS,
-      gasOracleType: getGasOracles(chain),
-      overhead: Object.fromEntries(
-        exclude(chain, supportedChainNames).map((remote) => [
-          remote,
-          multisigIsmVerificationCost(
-            defaultMultisigIsmConfigs[remote].threshold,
-            defaultMultisigIsmConfigs[remote].validators.length,
-          ),
-        ]),
-      ),
-      upgrade: core[chain].upgrade,
-    };
-  },
-);
+export const igp: ChainMap<IgpConfig> = objMap(owners, (chain, owner) => {
+  return {
+    owner,
+    oracleKey: DEPLOYER_ADDRESS,
+    beneficiary: KEY_FUNDER_ADDRESS,
+    gasOracleType: getGasOracles(chain),
+    overhead: Object.fromEntries(
+      exclude(chain, supportedChainNames).map((remote) => [
+        remote,
+        multisigIsmVerificationCost(
+          defaultMultisigIsmConfigs[remote].threshold,
+          defaultMultisigIsmConfigs[remote].validators.length,
+        ),
+      ]),
+    ),
+    upgrade: core[chain].upgrade,
+  };
+});
```

Comment:
nit: why is the formatting different?

Reply:
IgpConfig is shorter than OverheadIgpConfig